.github/workflows/update-version-test.yml

```diff
@@ -0,0 +1,41 @@
+name: Update Version
+
+on:
+  workflow_call:
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    env:
+      SHARED: /tmp/workdir/pki
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+
+      - name: Setup git
+        run: |
+          sudo apt-get -y install git
+          git config --global user.name "Dr. John Doe"
+          git config --global user.email jdoe@example.com
+
+      - name: Update to version with a phase
+        run: |
+          ./update_version.sh 12 1 1 beta1
```

Comment:
To be more realistic, this should be `12.1.0-beta1`. We create alpha/beta releases before the GA which always ends with `.0`. After GA, we only release updates like `.1`, `.2`, etc. without any phases.